src/session.ts

```diff
@@ -25,13 +25,16 @@ export type State =
 
 export interface Session {
   address: string
-  tokenBalance: any
+  tokenBalance: BigNumber
```

Comment:
I added the `BigNumber` type for `tokenBalance`, since `token.balanceOf` doesn't define its return type.
Here on line 58 and 84